default.nix

```diff
@@ -275,7 +275,8 @@ let overrideCabal = pkg: f: if pkg == null then null else haskellLib.overrideCab
         ########################################################################
         reflex = addFastWeakFlag (addReflexTraceEventsFlag (addReflexOptimizerFlag (self.callPackage (hackGet ./reflex) {})));
         reflex-dom = addReflexOptimizerFlag (doJailbreak reflexDom.reflex-dom);
-        reflex-dom-core = addReflexOptimizerFlag (doJailbreak reflexDom.reflex-dom-core);
+        reflex-dom-core = appendConfigureFlags (addReflexOptimizerFlag (doJailbreak reflexDom.reflex-dom-core))
+                                               (optional enableLibraryProfiling "-fprofile-reflex");
```

Comment:
seems like this should be a list or you meant to use the non list form `appendConfigureFlag`

Reply:
`appendConfigureFlags` is the list form, and `optional` should make it a list? Note the nixpkgs might be too old to include the one in the OP.

Maybe lets just worry about the `reflex` and `reflex-dom` PRs for now? Only once reunifcation is merged will we have a better idea whether its worth back-porting that nixpkgs commit.

Reply:
oh, right!